src/server/SocketServer/index.js

```diff
@@ -0,0 +1,49 @@
+const socketIo = require('socket.io');
+
+/**
+ * Abstract Api of socket.io lib
+ */
+class SocketServer {
```

Comment:
This API really looks like the `BaseModule` one, Houston, I think we got a problem 😝